src/current/_includes/releases/v23.1/v23.1.5.md

```diff
@@ -0,0 +1,93 @@
+
+<h3 id="v23-1-5-sql-language-changes">SQL language changes</h3>
+
+- Improved the inline documentation and error messages related to [`SHOW RANGES`](../v23.1/show-ranges.html). [#105278][#105278]
+- Allow inbound foreign keys on TTL tables. [#105341][#105341]
+- Added columns `default_value` and `origin` (default, override, external-override) to the [`SHOW CLUSTER SETTING`](../v23.1/show-cluster-setting.html) command. [#105452][#105452]
+- Added the `VIEWACTIVITY` and `VIEWACTIVITYREDACTED` [system privileges](../v23.1/security-reference/authorization.html#supported-privileges) for the `crdb_internal.cluster_queries` and `crdb_internal.node_queries` [tables](../v23.1/crdb-internal.html). [#105467][#105467]
+- Fixed the error message to be more understandable when attempting to create [UDFs](../v23.1/user-defined-functions.html) under a virtual or temporary schema. [#105627][#105627]
+
+<h3 id="v23-1-5-command-line-changes">Command-line changes</h3>
+
+- Running [`cockroch node decommission <nodeID>`](../v23.1/cockroach-node.html#node-decommission) for a node that has already been decommissioned will now exit with code 0, as had been the case in CockroachDB versions prior to v23.1.0. [#103933][#103933]
+
+<h3 id="v23-1-5-db-console-changes">DB Console changes</h3>
+
+- The [**Range Report** page](../v23.1/ui-hot-ranges-page.html#range-report) (route `/reports/range/:rangeID`) shows the **Hot Ranges** menu item as highlighted in the left side menu. The back button in the Range Report page redirects back to the Hot Ranges page. [#104980][#104980]
+- Added description to the tooltip for the `Idle` status only for the Active Transactions view. Excluded `Idle` status filter option for the Active Statements view. [#105062][#105062]
+
+<h3 id="v23-1-5-bug-fixes">Bug fixes</h3>
+
+- Added more precision to small percentage values on the percentage bars on the DB Console. [#105078][#105078]
+- Fixed a crash when using `DurationToNumber` with empty duration object on SQL Activity tables. [#105152][#105152]
+- `SpanStats` is no longer subject to stale information, and should be considered authoritative. [#105000][#105000]
+- Fixed a bug in MuxRangefeed implementation that may cause MuxRangefeed to become stuck if enough ranges encountered certain error concurrently. [#105186][#105186]
+- [`SHOW CHANGEFEED JOBS`](../v23.1/show-jobs.html) no longer fails on v22.2 / v23.1 mixed version clusters. [#105008][#105008]
+- [`GRANT SYSTEM ALL ...`](../v23.1/grant.html) no longer causes the grantee to be unable to login. This was due to a bug where `ALL` would include the `NOSQLLOGIN` system privilege. Since `NOSQLLOGIN` is the only "negative" privilege, it is now excluded from the `ALL` shorthand, and must be granted explicitly in order to restrict logins. [#105080][#105080]
```

Comment:
```suggestion
- [`GRANT SYSTEM ALL ...`](../v23.1/grant.html) no longer causes the grantee to be unable to log in. This was due to a bug where `ALL` would include the `NOSQLLOGIN` [system privilege](../v23.1/security-reference/authorization.html#supported-privileges). Since `NOSQLLOGIN` is the only "negative" privilege, it is now excluded from the `ALL` shorthand, and must be granted explicitly in order to restrict logins. [#105080][#105080]
```